apps/charterafrica/src/components/Tool/TechSkills.js

```diff
@@ -0,0 +1,49 @@
+import { RichTypography } from "@commons-ui/core";
+import { Box, Button } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+const TechSkills = React.forwardRef(function TechSkills(props, ref) {
+  const { list, title, sx } = props;
+  if (!list?.length) {
+    return null;
+  }
+  return (
+    <Box sx={sx} ref={ref}>
+      <RichTypography variant="p3" color="neutral.dark">
+        {title}
+      </RichTypography>
+      <br />
+      {list?.map((skill) => (
```

Comment:
```suggestion
      {list.map((skill) => (
```

At this point we know `list.length > 0` so `list.map` should exist.